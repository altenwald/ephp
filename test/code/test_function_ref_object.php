<?php

function &data() {
    $a = new stdClass();
    $a->data = 10;
    return $a;
}

$d = &data();
$e = &$d;
var_dump($e);

