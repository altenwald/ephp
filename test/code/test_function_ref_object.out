object(stdClass)#1 (1) {
  ["data"]=>
  int(10)
}
